### PR TITLE
Change sorting football matches and styling

### PR DESCRIPTION
**Pull request by synion**

### CODE REVIEW

File: app/models/round.rb [hunk 1]
```diff
@@ -18,4 +28,8 @@ def winner
          .order('round_points DESC', 'number_of_correct_bets DESC', 'goal_difference ASC')
          .first
   end
+
+  def self.current_round_id
```
Comment (kovson) [inline app/models/round.rb]: As I understand that correctly when there is no current round (like now, after group phase) the first round is displayed. I think that displaying the LAST round will be more logic there.

File: app/models/round.rb [hunk 2]
```diff
@@ -8,6 +10,14 @@ def user_score(user_id)
     bets.where(user_id: user_id).sum(:point)
   end
 
+  def self.page_current
+    default_page = 1
```
Comment (kovson) [inline app/models/round.rb]: Move it to const, please. 

Maybe ```current_page``` will be better than ```page_current``` what do you think?

File: app/views/football_matches/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <br />
-<% rounds.each do |round| %>
+<div id= 'header'>
+  <%= paginate rounds %>
```
Comment (kovson) [inline app/views/football_matches/index.html.erb]: Will it display pages? Am I right?

File: app/helpers/football_matches_helper.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+module FootballMatchesHelper
+  def first_url(url)
```
Comment (kovson) [inline app/helpers/football_matches_helper.rb]: First what? Please use more precisely names.

File: app/controllers/football_matches_controller.rb [hunk 5]
```diff
@@ -1,12 +1,17 @@
...
   end
 
   def create
     ImportMatches.call
     redirect_to root_path
   end
+
+  private
+
+  def current_round
```
Comment (kovson) [inline app/controllers/football_matches_controller.rb]: So current round or current page? Why ```current_round_id``` isn't used for current round?

File: app/views/kaminari/_first_page.html.erb [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<li class="page-item">
+  <%= link_to_unless current_page.first?, raw(t 'views.pagination.first'), first_url(url), remote: remote, class: 'page-link'  %>
```
Comment (kovson) [inline app/views/kaminari/_first_page.html.erb]: See comment about ```first_url``` above.

File: spec/views/football_matches/index.html.erb_spec.rb [hunk 7]
```diff
@@ -16,6 +16,9 @@
         match_date: DateTime.new(2017, 9, 20, 14, 30).in_time_zone('Warsaw')
       )
 
+      rounds.stub(:current_page).and_return(1)
```
Comment (kovson) [inline spec/views/football_matches/index.html.erb_spec.rb]: Allow to receive and return, please. Same to all ```stub``` methods.

File: app/views/football_matches/index.html.erb (hunk 3 above)
Comment (synion) [reply app/views/football_matches/index.html.erb]: yes it displays pagination

**Pull request merged**